### ISSUE TITLE
feature suggestion: enlarge ui or icon

### ISSUE THREAD

**Issue by zzm88** [kvakulo/Switcheroo#155]
It's a great tool, but when I use it on my 27 4k monitor, it seems too small. Hope there is a option to enlarge the whole ui, or just the icon of each process ( better)

Thanks to the author for this great tool. I've been searching a solution for years to replace the orginal alt+tab

**Comment by AaronCompNetSys** [comment]
Yes, it is fairly small on my 4k monitor as well and my eyes are not getting any younger.  Just setting the DPI for the OS would be great, as it looks like it already scales some.

**Comment by coezbek** [comment]
Hey guys, in version 0.9.5 of my fork I have merged DPI-aware scaling of Switcheroo. So if you use 150% or 200% display then Switcheroo now also uses double the screen area: https://github.com/coezbek/switcheroo
